3.10/release-notes-new-features310.md

```diff
@@ -13,8 +13,16 @@ Native ARM Support
 ------------------
 
 ArangoDB is now available for the ARM architecture, in addition to the x86-64
-architecture. It can run on Apple silicon (e.g. M1 chips) without emulation.
-It also runs on other ARM chips under Linux and Windows for ARM.
+architecture.
+
+It can natively run on Apple silicon (e.g. M1 chips). It was already possible to
+run 3.8.x and older versions on these systems via Rosetta 2 emulation, but not
+3.9.x because of its use of AVX instructions, which Rosetta does not emulate.
```

Comment:
3.9.x because of its use of AVX instructions, which Rosetta **2** does not emulate.